### PR TITLE
DriverAPI: add isAutoDepthResolveSupported query.

### PR DESCRIPTION
**Pull request by prideout**

### CODE REVIEW

File: filament/src/details/View.h [hunk 1]
```diff
@@ -217,6 +217,8 @@ class FView : public View {
     }
 
     void setRenderTarget(FRenderTarget* renderTarget) noexcept {
+        assert_invariant(!mMultiSampleAntiAliasingOptions.enabled ||
+                !renderTarget->hasSampleableDepth());
```
Comment (pixelflinger) [inline filament/src/details/View.h:221]: I think we could be setting a `nullptr` renderTarget here (e.g. to "unset" it).

Comment (prideout) [reply filament/src/details/View.h]: Excellent catch, fixed.

**Comment by prideout** [comment]
@pixelflinger can you point me in the right direction for the next step?  We need to use this query from PostProcessManager, right?

**Comment by prideout** [comment]
@pixelflinger this one is ready to go, I think.

**Review by pixelflinger — APPROVED** [review]

**Pull request merged**